build/conf/local.conf

```diff
@@ -1,4 +1,8 @@
-IMAGE_INSTALL_append = " python3-modules python3-robot-shared-data python3-robot-api python3-update-server python3-notify-server"
+IMAGE_INSTALL_append = " python3-robot-server python3-robot-shared-data python3-update-server python3-notify-server python3-robot-api robotserversystemd python3-uvicorn"
+DISTRO_FEATURES_append = " systemd"
+DISTRO_FEATURES_BCKFILL_CONSIDERED += "sysvinit"
```

Comment:
typo?
```suggestion
DISTRO_FEATURES_BACKFILL_CONSIDERED += "sysvinit"
```

Reply:
Also do we actually want backfill here? There's a lot of stuff that entirely relies on systemd.

Reply:
Think this is to keep sysvinit from getting backfilled.. if a distro feature isn't set, backfilled ones get added automatically. this is because we added new features and made them default.  if we don't want it, we can add it to DISTRO_FEATURES_BACKFILL_CONSIDERED to say that we know it would  be backfilled, but we don't want it.

Reply:
oh, i see, the semantics are "i personally considered (and rejected) this, bitbake you don't have to"